packages/orchestration/README.md

```diff
@@ -229,6 +219,28 @@ const response = await orchestrationClient.chatCompletion({
 return response.getContent();
 ```
 
+### Custom Request Configuration
+
+Set custom request configuration in the `requestConfig` parameter when calling `chatCompletion` function.
```

Comment:
```suggestion
Set custom request configuration in the `requestConfig` parameter when calling `chatCompletion()` method.
```

[styleguide] Use [Code Format for Method Names](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-code-format-for-method-names) and [Use Function vs Method Accordingly](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-function-vs-method-accordingly)